pkg/controller/cloudmetrics/cloudmetrics_controller.go

```diff
@@ -109,7 +111,25 @@ var postgresGaugeMetrics = []CroGaugeMetric{
 
 // redisGaugeMetrics stores a mapping between an exposed (redis) prometheus metric and multiple cloud provider specific metric
 // to add any addition metrics simply add to this mapping and it will be scraped and exposed
-var redisGaugeMetrics []CroGaugeMetric
+var redisGaugeMetrics = []CroGaugeMetric{
+	{
+		Name: redisMemoryUsagePercentage,
+		GaugeVec: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: redisMemoryUsagePercentage,
+				Help: "The percentage of redis used memory. Units: Bytes",
+			},
+			labels),
+		ProviderType: map[string]providers.CloudProviderMetricType{
+			providers.AWSDeploymentStrategy: {
+				PromethuesMetricName: redisMemoryUsagePercentage,
```

Comment:
Should we also be exposing `FreeableMemory` not just `MemoryUsage`

Reply:
no reason why we couldn't